mockserver.js

```diff
@@ -60,11 +61,73 @@ var parse = function (content) {
         }
     });
 
-    body = bodyContent.join('\n');
+    body = applyHooks(bodyContent).join('\n');
```

Comment:
Hi @odino, very good idea, it's a shame you did not mention it before. Unfortunately I don't have more time to spend. If you're not happy with what we did, I will keep using my fork.